### PR TITLE
F/wep 88 hide authors

### PR DESCRIPTION
**Pull request by penk85**
Added option for publisher to hide authors of an article. This allows for authors to stay private.
Updated db schema and queries with new category, and set the public API to validate whether authors are hidden before returning article data. Editor was updated to allow toggling changes.

### CODE REVIEW

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks very good 👍🏾

File: packages/api-db-mongodb/src/migration.ts [hunk 1]
```diff
@@ -179,7 +179,7 @@ export const Migrations: Migration[] = [
       function mapArticleTeaserGridBlock(block: any) {
         if (block.type === 'articleTeaserGrid') {
           return {
-            type: 'teaserGrid',
+            type: 'graphqlteaserGrid',
```
[the remark this answers is not in the document]
Comment (penk85) [reply packages/api-db-mongodb/src/migration.ts:182]: Huh. Good catch.

**Review by roosnic1 — APPROVED** [review]
🎉

**Review by faebebin — APPROVED** [review]
Looks good.

**Review by 0815pascal — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]

**Pull request merged**